src/app/components/Card/Card.module.css

```diff
@@ -1,18 +1,21 @@
 .card {
-  display: flex;
+  display: grid;
+  grid-template-columns: 1fr;
```

Comment:
Is this necessary?

Reply:
Yes. As a flexbox fanboy... but yes it is. With flexbox there was a problem with the image that was too big.